portfolio.html

```diff
@@ -62,14 +62,14 @@ <h1>Harry Cusworth</h1>
 				<p class="status">Aspirational Developer
 				</p>
 			</header>
-			
+					
 			<section class=contactDeets>
 					<h3>Contact Details</h3>
-					<div><i class="fas fa-globe" style="color:#B4535A"></i><a href=https://harrycusworth.com/> Consultancy Home Page</a></div>
-					<div><i class="far fa-envelope" style="color:#B4535A"></i> <a href=mailto:harry.cusworth@protonmail.com> E-Mail</a></div>
-					<div><i class="fas fa-phone" style="color:#B4535A"></i>  0792*********</div>
-					<div><i class="fab fa-linkedin" style="color:#B4535A"></i><a href=https://www.linkedin.com/in/harry-cusworth-83b56a182/> Linkedin</a></div>
-					<div><i class="fab fa-github" style="color:#B4535A"></i> <a href=https://github.com/HarryCusworth>GitHub</a></div>
```

Comment:
styles are inline within html and are all the same color. You could create a class selector ".far fab fas: { color:#B4535A} to solve this. Also hrefs are missing "..."